pkg/controller/context.go

```diff
@@ -99,26 +115,26 @@ func TemplateMapFromContext(ctx context.Context) TemplateMap {
 	if !ok {
 		return make(TemplateMap)
 	}
+
 	return m
 }
 
-// WithUser sets the user in the context.
-func WithUser(ctx context.Context, u *database.User) context.Context {
-	return context.WithValue(ctx, contextKeyUser, u)
+// WithUser stores the current user on the context.
+func WithUser(ctx context.Context, app *database.User) context.Context {
```

Comment:
The variable u makes more sense here than app?

Reply:
You may have caught me copy-pasting 😄